__tests__/app.test.js

```diff
@@ -44,6 +45,37 @@ describe("/api/topics", () => {
     })
 })
 
+describe("/api/articles", () => {
+    test("GET: 200 responds with an array of all articles with correct properties, including 'comment_count'", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+            expect(Array.isArray(response.body.articles)).toBe(true);
```

Comment:
This is probably not necessary. The other expects implicitly confirming that it is an array